spec/features/unsubscribe_all_spec.rb

```diff
@@ -0,0 +1,69 @@
+RSpec.feature "Bulk unsubscribe after receiving confirmation link" do
+  include GdsApi::TestHelpers::EmailAlertApi
+  include TokenHelper
+
+  scenario do
+    given_i_have_multiple_subscriptions
+    when_i_visit_the_manage_my_subscriptions_page
+    and_i_click_on_unsubscribe_from_everything
+    and_i_confirm_to_unsubscribe
+    then_i_can_see_i_have_been_unsubscribed
+  end
+
+  def given_i_have_multiple_subscriptions
+    @address = "test@test.com"
+    @subscriber_id = 123
```

Comment:
Minor: I notice we use `SecureRandom` in the other specs. Any reason to hard code it?

Reply:
No, just an oversight, I've changed to use `SecureRandom`.